src/super_gradients/training/datasets/segmentation_datasets/pascal_voc_segmentation.py

```diff
@@ -174,6 +175,12 @@ def _get_pascal_labels(self) -> np.ndarray:
             ]
         )
 
+    def output_image_shape(self) -> Tuple[int, int]:
```

Comment:
I'm not quite following why we introduce functions with hard-coded return values here?

Reply:
most segmentation recipes have no transforms (manning no processing as well) - there's a hidden assumption that all inputs are of the same size.  
The prediction inputs might be with different sizes and we need to be able to run segmentation prediction on them. in addition the predict input size might be different from the input size the model trained on.
with this property we make sure to resize all prediction-inputs to the same size as the training data